test/unit/SwapOperator/swapEnzymeVaultShareForETH.js

```diff
@@ -0,0 +1,170 @@
+const { ethers } = require('hardhat');
+const { expect } = require('chai');
+const { ETH } = require('../../../lib/constants').Assets;
+
+const {
+  utils: { parseEther },
+} = ethers;
```

Comment:
same here, probably fits on one line

Reply:
Nested decontructuring is automatically formatted like this by prettier. The only way I found to avoid having this multilined by prettier is to write it like this:
`const { parseEther } = ethers.utils;`